src/main/java/tc/oc/tablist/TabRender.java

```diff
@@ -53,6 +54,9 @@ private BaseComponent getContent(TabEntry entry, int index) {
   }
 
   private void appendAddition(TabEntry entry, int index) {
+    if (ViaUtils.getProtocolVersion(this.view.getViewer()) < ViaUtils.VERSION_1_8) {
+      return;
+    }
```

Comment:
This check shouldn't be here, instead, this renderer shouldn't be called for 1.7 clients